### PR TITLE
Fix recursive call in sourceDirectory

### PR DESCRIPTION
**Pull request by AntonKrasikov**
modifiedOnly parameter is not passed properly into recursive call. Recursive call uses default value (TRUE), so any files in sub-directories are not sourced when sourceDirectory(modifiedOnly=FASE) is run for the second time.

### CODE REVIEW

File: tests/sourceDirectory.R [hunk 1]
```diff
@@ -0,0 +1,11 @@
+library("R.utils")
+
+x = 1
```
Comment (HenrikBengtsson) [inline tests/sourceDirectory.R]: Coding style: Please use `x <- 1`

File: tests/sourceDirectory.R [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+# Test that modifiedOnly works for recursive subdirectories
+
+sourceDirectory('sourceDirectoryDir/', modifiedOnly = F)
```
Comment (HenrikBengtsson) [inline tests/sourceDirectory.R]: Robustness/Correctness: please use `FALSE` - not `F`

File: tests/sourceDirectoryDir/secondLevelDir/increment.R [hunk 3]
```diff
@@ -0,0 +1 @@
+x = x + 1
```
Comment (HenrikBengtsson) [inline tests/sourceDirectoryDir/secondLevelDir/increment.R]: Same here

**Comment by AntonKrasikov** [comment]
Fixed based on your comments.
Thanks for the feedback!

**Comment by HenrikBengtsson** [comment]
Thanks

**Comment by HenrikBengtsson** [comment]
FYI, R.utils 2.9.0, with this fix, is now on CRAN.

**Comment by AntonKrasikov** [comment]
Awesome, thanks for the prompt release!

**Pull request merged**